core/controllers/contributor_dashboard_admin_test.py

```diff
@@ -59,23 +59,23 @@ def setUp(self) -> None:
             self.get_user_id_from_email(self.TRANSLATION_ADMIN_EMAIL),
             feconf.ROLE_ID_TRANSLATION_ADMIN)
 
-    def test_non_admin_access_page_raise_404(self) -> None:
+    def test_non_admin_access_page_raise_401(self) -> None:
         self.login(self.USER_A_EMAIL)
         self.get_html_response(
-            '/contributor-dashboard-admin', expected_status_int=401)
+            '/contributor-admin-dashboard', expected_status_int=401)
```

Comment:
Instead of hardcoding, please use `feconf.CONTRIBUTOR_DASHBOARD_ADMIN_URL`, ditto for below.

Reply:
PTAL @lkbhitesh07